tasks/install_from_nodesource_debs.yml

```diff
@@ -0,0 +1,27 @@
+- name: Install HTTPS transport for APT
+  apt: name=apt-transport-https state=installed
+
+- name: Check if your distribution is supported
+  uri: url=https://deb.nodesource.com/node/dists/{{ansible_distribution_release}}/Release
```

Comment:
I would rather pull from https://nodejs.org/dist/. NVM does this. This would work better across distros.